### PR TITLE
Add /8ball command

### PR DESCRIPTION
**Pull request by mctaylors**
@neroduckale was bored so I made this feature.

### CODE REVIEW

**Comment by Apceniy** [comment]
Why are the answers in tt-ru locale ordered this way?

**Comment by mctaylors** [comment]
> Why are the answers in tt-ru locale ordered this way?

`mctaylors-ru`, to be correct. It's my language and I translate it as I wish. If I'm happy with the translation, then that's the way it's going to be *or until the maintainer himself says so*.

File: locale/Messages.ru.resx [hunk 1]
```diff
@@ -594,4 +594,64 @@
...
+      <value>Спроси позже</value>
+  </data>
+  <data name="EightBallNeutral3" xml:space="preserve">
+      <value>Лучше не рассказывать</value>
+  </data>
+  <data name="EightBallNeutral4" xml:space="preserve">
+      <value>Сейчас нельзя предсказать</value>
+  </data>
+  <data name="EightBallNeutral5" xml:space="preserve">
+      <value>Сконцентрируйся и спроси опять</value>
```
Comment (Octol1ttle) [inline locale/Messages.ru.resx:640]:
```suggestion
      <value>Сконцентрируйся и спроси снова</value>
```

File: src/Commands/ToolsCommandGroup.cs [hunk 2]
```diff
@@ -496,4 +496,65 @@ private Task<Result> SendTimestampAsync(TimeSpan? offset, IUser executor, Cancel
...
+    /// <param name="question">Unused input.</param>
+    /// <remarks>
+    ///     The 8-Ball answers were taken from <a href="https://en.wikipedia.org/wiki/Magic_8_Ball#Possible_answers">Wikipedia</a>.
+    /// </remarks>
+    /// <returns>
+    ///     A feedback sending result which may have succeeded.
+    /// </returns>
+    [Command("8ball")]
+    [DiscordDefaultDMPermission(false)]
+    [Description("Ask a question to")]
```
Comment (Octol1ttle) [inline src/Commands/ToolsCommandGroup.cs:512]: Ask a question to... what?

File: src/Commands/ToolsCommandGroup.cs [hunk 3]
```diff
@@ -496,4 +496,65 @@ private Task<Result> SendTimestampAsync(TimeSpan? offset, IUser executor, Cancel
...
+    private Task<Result> AnswerEightBallAsync(IUser bot, CancellationToken ct)
+    {
+        var typeNumber = Random.Shared.Next(0, 4);
+        var embedColor = typeNumber switch
+        {
+            0 => ColorsList.Blue,
+            1 => ColorsList.Green,
+            2 => ColorsList.Yellow,
+            3 => ColorsList.Red,
+            _ => ColorsList.Default
```
Comment (Octol1ttle) [inline src/Commands/ToolsCommandGroup.cs:549]: Should throw instead, indexes other than 0-3 will lead to an IndexOutOfBoundsException

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: src/Commands/ToolsCommandGroup.cs [hunk 4]
```diff
@@ -496,4 +496,65 @@ private Task<Result> SendTimestampAsync(TimeSpan? offset, IUser executor, Cancel
 
         return _feedback.SendContextualEmbedResultAsync(embed, ct: ct);
     }
+
+    /// <summary>
+    ///     A slash command that shows a random magic 8-ball's answer.
```
Comment (Octol1ttle) [inline src/Commands/ToolsCommandGroup.cs:501]:
```suggestion
    ///     A slash command that shows a random answer from the Magic 8-Ball.
```

File: src/Commands/ToolsCommandGroup.cs [hunk 5]
```diff
@@ -496,4 +496,65 @@ private Task<Result> SendTimestampAsync(TimeSpan? offset, IUser executor, Cancel
...
+    /// </summary>
+    /// <param name="question">Unused input.</param>
+    /// <remarks>
+    ///     The 8-Ball answers were taken from <a href="https://en.wikipedia.org/wiki/Magic_8_Ball#Possible_answers">Wikipedia</a>.
+    /// </remarks>
+    /// <returns>
+    ///     A feedback sending result which may have succeeded.
```
Comment (Octol1ttle) [inline src/Commands/ToolsCommandGroup.cs:508]:
```suggestion
    ///     A feedback sending result which may or may not have succeeded.
```

File: src/Commands/ToolsCommandGroup.cs [hunk 6]
```diff
@@ -496,4 +496,65 @@ private Task<Result> SendTimestampAsync(TimeSpan? offset, IUser executor, Cancel
...
+    /// </returns>
+    [Command("8ball")]
+    [DiscordDefaultDMPermission(false)]
+    [Description("Ask the magic 8-ball a question")]
```
Comment (Octol1ttle) [inline src/Commands/ToolsCommandGroup.cs:512]:
```suggestion
    [Description("Ask the Magic 8-Ball a question")]
```

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

**Review by Octol1ttle — APPROVED** [review]

**Pull request merged**